sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -449,14 +449,17 @@ with Text Analytics and require Python 3.5 or later.
 Authenticate the client with a Cognitive Services/Text Analytics API key or a token credential from [azure-identity][azure_identity]:
 * [sample_authentication.py][sample_authentication] ([async version][sample_authentication_async])
 
-In a batch of documents:
+Common scenarios
```

Comment:
It will be good to also add information about OM in the Analyze Sentiment section.
i.e. Java => https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/textanalytics/azure-ai-textanalytics#analyze-sentiment